### DISCUSSION TITLE
Zoom Feature behaves differently in Thiea and VS-Code

### DISCUSSION THREAD

**Discussion by rsoika** [eclipse-glsp/glsp#1145] [Q&A]
When working in a GLSP diagram you can zoom the diagram in and out using the mouse wheel. I am still not clear if this is a browser/svg thing or if there is an additional GLSP code involved.

But I recognized that the zoom feature is working on Theia very well and you can increase and reduce the size of the diagram in a fine-grained way. However, the same thing doesn't behaves that way in vs-code. There, the zoom steps are much coarser. 

What could be the reason for this different behaviour? And is there a way to influence the size of the zoom steps somhow in GLSP?

Theia:

![Peek 2023-10-17 19-05](https://github.com/eclipse-glsp/glsp/assets/1339124/a0efa3d8-c431-4c82-a2c6-5d0963cb115e)

VS-Code:

![Peek 2023-10-17 19-04](https://github.com/eclipse-glsp/glsp/assets/1339124/84701fe3-d1a1-4cd2-a226-90ec4fb4efc1)

**Comment by tortmayr** [comment]
@rsoika Thanks for raising I have opened https://github.com/eclipse-glsp/glsp/issues/1152 to track this
